### PR TITLE
Feature/UI updates

### PR DESCRIPTION
**Pull request by chasefleming**
Addresses: https://github.com/onflow/fcl-discovery/issues/81, https://github.com/onflow/fcl-discovery/issues/149

--------

Screen Shot 2023-01-20 at 11 11 27 AM

Screen Shot 2023-01-20 at 11 10 19 AM

### CODE REVIEW

File: components/Headers/FlowHeader.js [hunk 1]
```diff
@@ -1,41 +1,12 @@
...
-        <AppLogo src="/logo.svg" alt="Flow Logo" />
-      </AppLogoWrapper>
-      <AppTitle>Choose a Provider</AppTitle>
-    </AppHeader>
+    <Center>
+      <Flex flexDirection="column" alignItems="center" marginBottom={4}>
+        <Image src="/logo.svg" alt="Logo" w="140px" h="auto" marginBottom={3} />
+        <Text color="grey">Choose a Provider</Text>
+      </Flex>
+    </Center>
```
Comment (justinbarry) [inline components/Headers/FlowHeader.js:10]: Incredible how much cleaner this is!

File: components/Headers/__tests__/__snapshots__/AppHeader.test.js.snap [hunk 2]
```diff
@@ -1,36 +1,48 @@
...
-  <div
-    class="sc-eCYdqJ hBueAj"
-  />
 </div>
 `;
 
 exports[`Component: AppHeader should render the the component with icon 1`] = `
 <div
-  class="sc-bczRLJ dIvpJ"
+  class="chakra-stack css-184jys2"
```
Comment (justinbarry) [inline components/Headers/__tests__/__snapshots__/AppHeader.test.js.snap:26]: Are these CSS classes stable?

**Review by justinbarry — COMMENTED** [review]
Really nice work! Much easier to read now.

**Comment by justinbarry** [comment]
@chasefleming Is there a way to test this without spinning up something locally?

**Review by gregsantos — COMMENTED** [review]
It looks much cleaner and think it an overall improvement!
I know we are working with minimal space but I feel it needs some padding increases around the main Discovery window to feel less "tight".  (Especially on the top/bottom)
I'd increase the gap between buttons as well.

I think you can remove the "Installed" chip and just show "install" chip if it's not available/installed.
The Left alignment seems off to me and I'd suggest Center aligned Title and host.

File: components/Headers/__tests__/__snapshots__/AppHeader.test.js.snap (hunk 2 above)
Comment (chasefleming) [reply components/Headers/__tests__/__snapshots__/AppHeader.test.js.snap:26]: @justinbarry good catch. I added something to make the classes generated by emotion for this deterministic.

**Review by justinbarry — APPROVED** [review]
LGTM

**Pull request merged**